app/partials/buy-sell.jade

```diff
@@ -13,6 +13,7 @@
                         modal-open="status.modalOpen"
                         change-currency="changeCurrency(currency)"
                         limits="limits"
+                        can-trade="exchange.profile.canTrade"
```

Comment:
`exchange.profile.canTrade` is undefined (falsy) when the user doesn't have an account or when the `canTrade` field is missing. A "negatrive" attribute would be safer here, e.g. `trade-suspended-until`.